### PR TITLE
restock query implement and test

### PR DESCRIPTION
**Pull request by joanne3634**

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -131,6 +131,26 @@ def all(cls):
...
+    def find_by_restock(cls, restock):
+        """ Returns all of the Inventory that quantity lower than their restock level
+        Args:
+            restock (boolean): if restock is true than return the list, if false than return normal list all
+        """
+        cls.logger.info('Processing quantity < restock_level query ...')
+        if restock is True:
+            return cls.query.filter(cls.quantity < cls.restock_level)
+        else:
+            return cls.query.all()
```
Comment (faberald) [inline service/models.py]: Why we return all if restock == false?

Comment (joanne3634) [reply service/models.py]: original I design restock==false as not setting restock, but I think you are right, quantity >= restock_level make more sense, fix it already

**Comment by SonakshiRaizada** [comment]
Also, why "db" is changed to "DB"? Is it some standard format that we should follow?

**Comment by joanne3634** [comment]
> What is the pylintrc used for?
> I thought we already have a .pylintrc file to check the pylint score, are you adding more constraints?

yes, if you modify model.py and try to commit through cmd inside vagrant env, the pylint check the pre-commit and would not pass the pylint. The error would be corrected by Line 368 of pylintrc `generated-members=Column,Integer,Boolean,String`, if I do not add this constraints, then it will not allow to commit.

**Comment by joanne3634** [comment]
> Also, why "db" is changed to "DB"? Is it some standard format that we should follow?

@SonakshiRaizada this is the pylint standard, since I should pass the pylint then I can add a commit.

File: tests/test_service.py [hunk 2]
```diff
@@ -84,6 +84,43 @@ def test_get_inventory_list(self):
...
+                            query_string='restock=false')
+
+    def test_query_inventory_list_by_restock_level(self):
+        """ Query Inventories by restock_level """
+        inventories = []
+        for _ in range(0,2):
+            test = Inventory(product_id=_, quantity=_, restock_level=20)
+            test.save()
+            inventories.append(test)
+        for _ in range(2,5):
```
Comment (eatingzhang0611) [inline tests/test_service.py]: Not following pylint rules.
test_service.py:105:24: C0326: Exactly one space required after comma.

**Pull request merged**